st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -181,6 +181,12 @@
                                 'doesn\'t finish in timeout seconds.'),
                 'type': 'integer',
                 'default': FABRIC_RUNNER_DEFAULT_ACTION_TIMEOUT
+            },
+            'port': {
```

Comment:
Added support in https://github.com/lakshmi-kannan/st2/commit/e93bb0002d1c34ae90dbc080f56571b01d6beb9b

Examples: https://gist.github.com/lakshmi-kannan/b6f58884391fcade6aad

I still left the port parameter in tact. If the hosts are all running on non-standard port, having a port param seems useful (less typing).